app/models/nomis/sentence_detail.rb

```diff
@@ -8,16 +8,25 @@ class SentenceDetail
                   :home_detention_curfew_eligibility_date,
                   :home_detention_curfew_actual_date,
                   :parole_eligibility_date,
-                  :post_recall_release_date,
-                  :post_recall_release_override_date,
                   :release_date,
                   :sentence_start_date,
-                  :tariff_date
+                  :tariff_date,
+                  :sentence_expiry_date,
+                  :licence_expiry_date,
+                  :actual_parole_date
 
     attr_writer :automatic_release_date,
                 :automatic_release_override_date,
                 :conditional_release_date,
-                :conditional_release_override_date
+                :conditional_release_override_date,
+                :post_recall_release_date,
+                :post_recall_release_override_date
+
+    def initialize(fields = {})
```

Comment:
Would it be good to use this in the production code that initializes this object? ([on line 49 of this same file, below](https://github.com/ministryofjustice/offender-management-allocation-manager/pull/1146/files#diff-af383291a16f3cffa1cb19f273fdb104R49))

I think it's good to pick one strategy (I don't have strong opinions around custom initializers vs `#tap`)

Reply:
I think the other PR changes this as you suggest - It was part of a refactoring, so I didn't include it here